pyiron_base/generic/hdfio.py

```diff
@@ -893,16 +843,6 @@ def _get_h5_path(self, name):
         """
         return posixpath.join(self.h5_path, name)
 
-    def _get_store_node(self, store):
-        """
-        Index store with the current h5_path set, but avoid to index by '/', because that raises an IndexError.
```

Comment:
This is not an issue when directly using the `h5py.File` interface, right?

Reply:
No, `File`s are also `Group`s, so you can index them directly.